app/controllers/idv/document_capture_controller.rb

```diff
@@ -1,5 +1,6 @@
 module Idv
   class DocumentCaptureController < ApplicationController
+    include AcuantConcern
```

Comment:
Yeah, I think I prefer 1 layer of concern inclusion, and trying to avoid "utilities" if we can name things more precisely. As it stands, the StepUtilitiesConcern is included in several places--would it make sense to make that change / refactor as a separate PR after this lands?